WordPress/Classes/ViewRelated/Blog/QuickStartChecklistCell.swift

```diff
@@ -53,15 +67,38 @@ class QuickStartChecklistCell: UITableViewCell {
         }
     }
 
+    public var topSeparatorIsHidden: Bool = false {
+        didSet {
+            topSeparator?.isHidden = topSeparatorIsHidden
+        }
+    }
+
 
     override func awakeFromNib() {
         super.awakeFromNib()
 
         if Feature.enabled(.quickStartV2) {
-            WPStyleGuide.configureLabel(titleLabel, textStyle: .headline)
-            WPStyleGuide.configureLabel(descriptionLabel, textStyle: .subheadline)
+            setupConstraints()
         }
     }
 
     static let reuseIdentifier = "QuickStartChecklistCell"
 }
+
+extension QuickStartChecklistCell {
```

Comment:
I think this should be a `private extension`?

Reply:
Yes you know that I like that solution but then I saw this: https://google.github.io/swift/#access-levels
What do you think?

Reply:
I definitely think this is something that's inconsistent in our code base. I'm fine with you picking whichever you prefer for this PR! There's a `private extension` added in `QuickStartListTitleCell.swift`, so just ensure these two are consistent?